index.html

```diff
@@ -197,6 +197,7 @@
             <h4>[[title]]</h4>
             <p class="repoLink"><a href="[[repoURL]]" target="_blank">[[repoTitle]]</a></p>
             <p>[[description]]</p>
+            <p><b>Topics:</b><br/>[[topics]]</p>
```

Comment:
might be good to visualize this with a tag-like layout, e.g. a div with a certain class. Here is a suggestion:

``` html
<span class="pill">Topic</span>
```
``` css
.pill {
    border-radius: 2em;
    padding: 2px 10px;
    background-color: lightblue;
}
```

Reply:
![topic pills](https://user-images.githubusercontent.com/6791333/105686100-2411a000-5ef7-11eb-926a-5c5718585e0b.png)

Reply:
Nice! The styling looks a bit funky when the repo has a lot of topics but it is certainly an improvement already!

![Screen Shot 2021-01-25 at 22 14 06 ](https://user-images.githubusercontent.com/163029/105767319-37565700-5f5b-11eb-9177-1e86bb3503c4.JPG)

Reply:
it may still need some margin-bottom (5px or so should be fine) :-)